hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -129,98 +108,15 @@ public void storeValidScmCertificate(BigInteger serialID,
   public void checkValidCertID(BigInteger serialID) throws IOException {
     lock.lock();
     try {
-      if ((getCertificateByID(serialID, VALID_CERTS) != null) ||
-          (getCertificateByID(serialID, CertType.REVOKED_CERTS) != null)) {
+      if ((getCertificateByID(serialID) != null) ||
```

Comment:
Minor: we are repeating the same check twice here

Reply:
you are right, thank you for spotting this one...